zwavejs2mqtt.json

```diff
@@ -0,0 +1,43 @@
+{
+    "name": "zwavejs2mqtt",
+    "release": "12.1-RELEASE",
+    "artifact": "https://github.com/tprelog/iocage-zwavejs2mqtt.git",
+    "official": "false",
+    "devfs_ruleset": {
+        "paths": {
+            "zfs": null,
+            "bpf*": null,
+            "cua*": null
+        },
+        "includes": [
+            "1",
+            "2",
+            "3"
+        ]
+    },
+    "properties": {
+        "allow_raw_sockets": "1",
+        "dhcp": "1"
+    },
+    "pkgs": [
+        "bash",
+        "ca_root_nss",
+        "curl",
+        "git-lite",
+        "node",
+        "npm",
+        "python",
+        "rsync",
+        "wget"
+    ],
+    "packagesite": "http://pkg.FreeBSD.org/${ABI}/quarterly",
```

Comment:
Quarterly packages intentional here?

Reply:
I just assumed `quarterly` would be more stable but I can use `latest` if that's what's recommended

zwavejs2mqtt is downloaded from GitHub and built / installed using `npm` - This plugin provides a simple console menu that can be used to upgrade zwavejs2mqtt without needing to run a Plugin UPDATE

Reply:
Don't matter to me, just warning since a lot of folks find out later that quarterly really does mean quarterly, so security updates and whatnot may be slow arriving :)

Reply:
good point. I changed it